openapi.yml

```diff
@@ -57,6 +57,26 @@ paths:
           required: false
           schema:
             type: integer
+        - name: cocina_model_json
+          in: query
+          description: the JSON serialization of the Cocina object
+          required: false
+          content:
+            application/json:
+              schema:
+                type: object
+        - name: created_at
+          in: query
+          description: the creation date of the Cocina object
+          required: false
+          schema:
+            type: string
```

Comment:
`format: date-time`?

Reply:
thanks, was looking for something like that...